README.md

```diff
@@ -106,6 +106,12 @@
 | [RISCV_XPACK](https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack) |  -   |  -  |  -  |    -    | riscv-none-elf-gcc | riscv-none-elf-gcc |
 | [RISCV_NONE](https://archlinux.org/packages/extra/x86_64/riscv64-elf-gcc/) |  -   |  -  |  -  |    -    | riscv64-elf-gcc | riscv64-elf-gcc |
 
+8) Static Analysis
+
+Support [Coverity](https://scan.coverity.com/) Scan tool.
+
+See [nightly Coverity scanning](https://github.com/DMTF/libspdm/blob/main/.github/workflows/coverity.yml) in github CI.

```

Comment:
This has been disabled. I have to run it manually. You can delete this line.

Reply:
deleted